mln-lambda/README.md

```diff
@@ -0,0 +1,7 @@
+# MapLibre Native Lambda
+
+This directory contains the code of the serverless function that is used to gather results from continious benchmarking of MapLibre Native.
+
+It is an [AWS Lambda](https://en.wikipedia.org/wiki/AWS_Lambda) function written in TypeScript that uses an [Amazon DynamoDB](https://en.wikipedia.org/wiki/Amazon_DynamoDB) for storage. The source code lives in `lambda/index.ts`.
+
+It can be deployed using [AWS CDK](https://en.wikipedia.org/wiki/AWS_Cloud_Development_Kit) (see `./cdk.ts`) with the command `npm run cdk`.
```

Comment:
Would be great if you could add some image with the architecture of this, or a few more words on the motivation around it.

Are the results gathered while using the CI-runners or is this a general solution? benchmark of clients performance or CI pipelines...

Reply:
I can expand the README a bit, but in terms of architecture it is just an API gateway, AWS Lamba and DynamoDB table.

This setup is free (our usage is welll within free tier), maintenance free and intergrates well with other AWS services. Using AWS CDK is an TypeScript API that declaratively specifies infrastructure and allows for documented and hassle free (re-)deployment.

My setup is largely based on the output of `cdk init sample-app --language=typescript`.